### PR TITLE
Replace form link for a more User-friendly link.

### PR DESCRIPTION
**Pull request by metalvegetarianoprogresivo**
## What?
We changed the link to the form it so it looks better for the user.

## Why?
The current link to the form looks kind of ugly from a user-friendly perspective. 

## Proof
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - Selecciona uno de los retos.
 - Crea tu aplicación que cumpla con los criterios de aceptación del reto.
 - Sube tu app a GitHub
-- Crea un issue en este repositorio llenando el siguiente formulario -> [form](https://github.com/IntersysConsulting/talent-land-2019-challenges/issues/new)
+- Crea un issue en este repositorio llenando el este [formulario](https://github.com/IntersysConsulting/talent-land-2019-challenges/issues/new).
```
Comment (cmedinilla) [inline README.md]: `- Crea un issue en este repositorio llenando este [formulario](https://github.com/IntersysConsulting/talent-land-2019-challenges/issues/new).`

**Review by cmedinilla — CHANGES_REQUESTED** [review]
@metalvegetarianoprogresivo Good work, only a little comment

**Review by cmedinilla — CHANGES_REQUESTED** [review]
@metalvegetarianoprogresivo Good work, only a little comment

**Comment by metalvegetarianoprogresivo** [comment]
♻️ Done, re-review?

**Comment by cmedinilla** [comment]
LGTM

**Pull request merged**